sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+---
+friendly_name: Urlbar Clients Daily
+description: Daily per-client metrics related to urlbar usage
+owners:
+  - tbrooks@mozilla.com
```

Comment:
I would recommend having an additional DE as owner.  That can't be me since I'm leaving but I don't know who would be most appropriate

Reply:
This can be either me or @jklukas